python/README.md

```diff
@@ -82,18 +95,31 @@ response = client.request('mymethod', {'ping': True})
 print(response)
 ```
 
-Make an asynchronous request using `asyncio`:
+Make an asynchronous request, using the `AsyncClient` and `asyncio`:
 ```py
 import asyncio
-from rockets import Client
+from rockets import AsyncClient
 
-client = Client('myhost:8080')
+client = AsyncClient('myhost:8080')
 
 request_task = client.async_request('mymethod', {'ping': True})
 asyncio.get_event_loop().run_until_complete(request_task)
 print(request_task.result())
 ```
 
+If the `RequestTask` is not needed, i.e. no `cancel()` or `add_progress_callback()` is desired, use the `request()` coroutine:
```

Comment:
Maybe also mention somewhere that in notebooks users can simply use `await`?

Reply:
+1, also I would like to suggest the following:
- it would be nice to have a brief introduction as to why there is both a Client and AsyncClient
- I would move "Handle a request error" right after "Make a synchronous, blocking request" to avoid mixing async and non-async examples too much
- Rename "Send notifications" to "Send a notification to the server"
- some of the `import Client` should also be changed to `import AsyncClient`

Reply:
Done

Reply:
typo "to types of clients" should be "two types of clients", otherwise good for me!